src/cmake/usingOpenCV.cmake

```diff
@@ -1,5 +1,5 @@
-find_package(opencv REQUIRED)
-if (opencv_FOUND)
+find_package(OpenCV REQUIRED)
+if (OpenCV_FOUND)
   set_property(TARGET opencv::opencv PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${DIRECTORIES})
```

Comment:
Please add a quick note to `docs/source/Support/bskReleaseNotes.rst` to say

Fixed compiling Basilisk with ``opNav`` flag set to True on Linux platform

Reply:
If you can, could you squash the last 2 commits into one commit?  We've been trying to clean up the branches before moving to develop :-)